code/__DEFINES/construction.dm

```diff
@@ -9,6 +9,15 @@
 #define GIRDER_DISPLACED 3
 #define GIRDER_DISASSEMBLED 4
 
+//rwall construction states
+#define INTACT 0
+#define SUPPORT_LINES 1
+#define COVER 2
+#define CUT_COVER 3
+#define BOLTS 4
+#define SUPPORT_RODS 5
+#define SHEATH 6
```

Comment:
These names are a bit too generic for global defines. Should probably have an `RWALL_` prefix, similar to the other defines in this file.